base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -913,17 +916,16 @@ where B: BlockchainBackend
         if db.contains(&DbKey::BlockHash(block_hash))? {
             return Ok(BlockAddResult::BlockExists);
         }
-        // Perform orphan block validation.
-        if let Err(e) = self.validators.orphan.validate_internal_consistency(&block) {
```

Comment:
Not sure, this trait and function are not used - perhaps should be removed in this PR

Reply:
Checked that these validations are still done in the BlockBodyFullValidator::validate function, called in reorganize_chain when adding to the chain.

Reply:
Discussed offline, we are going to clean this up in another PR